### PR TITLE
Removed hesitancy code that was still left behind.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

_Description of the feature or fix with link to the issue if existing_

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/anniversary/partials/ReportedEvent.tsx [hunk 1]
```diff
@@ -14,8 +14,6 @@ function ReportedEvent({ reportedEvent }: IProps) {
         return 'loose-weight-1';
       case 'GENERAL':
         return 'location';
-      case 'HESITANCY':
```
Comment (selvach) [inline src/features/anniversary/partials/ReportedEvent.tsx:17]: We should keep this!

File: src/features/anniversary/types/index.ts [hunk 2]
```diff
@@ -6,7 +6,6 @@ export type TBadge =
   | 'TEST_RESULTS'
   | 'VACCINATION_STATUS'
   | 'VACCINE_SIDE_EFFECTS'
-  | 'HESITANCY'
```
Comment (selvach) [inline src/features/anniversary/types/index.ts:9]: We should keep this!

File: src/features/anniversary/partials/ReportedEvent.tsx (hunk 1 above)
Comment (aikewoody) [reply src/features/anniversary/partials/ReportedEvent.tsx:17]: That's what PR's are for! :)

File: src/features/anniversary/types/index.ts (hunk 2 above)
Comment (aikewoody) [reply src/features/anniversary/types/index.ts:9]: That's what PR's are for! 👍

**Pull request merged**